src/machine/board_nrf52840_generic.go

```diff
@@ -0,0 +1,27 @@
+//go:build nrf52840 && nrf52840_generic
+
+package machine
+
+const HasLowFrequencyCrystal = true
+
+var (
+	LED          = NoPin
+	SDA_PIN      = NoPin
+	SCL_PIN      = NoPin
+	UART_TX_PIN  = NoPin
+	UART_RX_PIN  = NoPin
+	SPI0_SCK_PIN = NoPin
+	SPI0_SDO_PIN = NoPin
+	SPI0_SDI_PIN = NoPin
+
+	// https://pid.codes/org/TinyGo/
+	usb_VID uint16 = 0x1209
+	usb_PID uint16 = 0x9090
```

Comment:
Where did this one come from? Did you allocate this yourself?

Reply:
I refactored this a bit so that it is configurable via build flag.